### PR TITLE
Keystone in Next.js example

### PR DESCRIPTION
**Pull request by flexdinesh**
- Example to demonstrate how to use Keystone within a Next.js application
- Deployable to vercel (tested)
- Seed data is setup during application boot. Lists are made readonly to prevent bad actors overloading the file db. 

Note: ~One click deploy to vercel will only work after `keystone-6/core` is release with next@13.~ Works now.

## TODO
- [x] Readme.md update to reflect the mental model clearly
- [x] Experiment: _One click deploy to vercel_ link

### CODE REVIEW

File: examples/keystone-in-nextjs/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Keystone in Next.js
+
+Keystone could double down as a powerful data engine within Next.js applications without having to host a separate Keystone server. This is made possible by Keystone's `getContext` API.
+
+- **CRUD data within your Next.js server**: You can use the Keystone data APIs directly in Next.js `getStaticProps` or `getServerSideProps` to CRUD data. ⚡️
+- **CRUD data from browser**: You can use the generated Keystone GraphQL schema to setup your own GraphQL server in a Next.js route. This enables you to send GraphQL requests from the browser. 🤯 (refer to [/pages/api/graphql-yoga.ts](/pages/api/graphql-yoga.ts) for implementation details)
```
Comment (borisno2) [inline examples/keystone-in-nextjs/README.md:6]:
```suggestion
- **CRUD data from browser**: You can use the generated Keystone GraphQL schema to setup your own GraphQL server in a Next.js route. This enables you to send GraphQL requests from the browser. 🤯 (refer to [/pages/api/graphql.ts](/pages/api/graphql.ts) for implementation details)
```

File: examples/keystone-in-nextjs/src/keystone/schema.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { list } from '@keystone-6/core';
+import { text, password, timestamp } from '@keystone-6/core/fields';
+import type { Lists } from '.keystone/types';
+
+const permissions = {
+  authenticatedUser: ({ session }: any) => !!session?.data,
+  public: () => true,
+  readOnly: {
+    operation: {
+      query: () => true,
```
Comment (borisno2) [inline examples/keystone-in-nextjs/src/keystone/schema.ts:10]:
```suggestion
      query: allowAll,
```

File: examples/keystone-in-nextjs/src/keystone/schema.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      create: () => false,
+      update: () => false,
+      delete: () => false,
```
Comment (borisno2) [inline examples/keystone-in-nextjs/src/keystone/schema.ts:13]: Can we use our `denyAll`, `allowAll` and `allOperations` functions instead (obviously this needs to be moved above `query`
```suggestion
      ...allOperations(denyAll)
```

File: examples/keystone-in-nextjs/README.md (hunk 1 above)
Comment (flexdinesh) [reply examples/keystone-in-nextjs/README.md:6]: Yup. I have some changes in local. Will push this along with that.

File: examples/keystone-in-nextjs/src/keystone/schema.ts (hunk 3 above)
Comment (flexdinesh) [reply examples/keystone-in-nextjs/src/keystone/schema.ts:13]: Yup. I have some changes in local. Will push this along with that.

File: examples/keystone-in-nextjs/src/keystone/schema.ts (hunk 2 above)
Comment (flexdinesh) [reply examples/keystone-in-nextjs/src/keystone/schema.ts:10]: Yup. I have some changes in local. Will push this along with that.

**Review by emmatown — APPROVED** [review]

**Review by borisno2 — APPROVED** [review]

**Pull request merged**